charts/em-npa/values.yaml

```diff
@@ -13,15 +13,18 @@ npa:
   livenessTimeout: 5
   livenessPeriod: 15
   livenessFailureThreshold: 10
+  aadIdentityName: em
   keyVaults:
-    rpa:
-      resourceGroup: rpa
+    em-npa:
+      resourceGroup: em-npa
       secrets:
         - AppInsightsInstrumentationKey
-    s2s:
-      resourceGroup: rpe-service-auth-provider
-      secrets:
         - microservicekey-em-npa-app
+#        - npa-POSTGRES-DATABASE
+#        - npa-POSTGRES-HOST
+#        - npa-POSTGRES-PASS
```

Comment:
You need to add this here and override in preview template so that u don't need to override in each environment all DB ones can be moved to like https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L6
and override them in preview like https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.preview.template.yaml#L6